### PR TITLE
Make code snippets longer than one line fenced code blocks

### PR DESCRIPTION
**Pull request by msmithcodat**
# Description

Please include a summary of the new content you're adding or the reason for the change. Please include relevant context.

## Type of change

Please delete options that are not relevant.

- [ ] New document(s)/updating existing
- [ ] Fixes
- [X] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/assess/metrics/accounting/api-marketing-metrics.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ View the Marketing Metrics [formulas](/assess/metrics/accounting/api-financial-m
 
 `GET /data/companies/{companyId}/connections/{connectionId}/assess/accountingMetrics/marketing`
 
-#Parameters
+# Parameters
```
Comment (d-laing) [inline docs/assess/metrics/accounting/api-marketing-metrics.md:35]: Good spot.

File: docs/assess/portal/categorization-of-accounts.md [hunk 2]
```diff
@@ -93,6 +93,8 @@ You can export the chart of accounts in an Excel file:
 The downloaded Excel file contains:
 
 - The **Categorized Account Balances** tab.
-- The output of
-  `GET /data/companies/{companyId}/connections/{connectionId}/assess/accounts/categories` endpoint in <a href="/assess-api#/operations/get-data-companies-companyId-connections-connectionId-assess-accounts-categories">API reference</a> listing the categories.
+- The output of the endpoint below in <a href="/assess-api#/operations/get-data-companies-companyId-connections-connectionId-assess-accounts-categories">API reference</a> listing the categories.
```
Comment (d-laing) [inline docs/assess/portal/categorization-of-accounts.md:96]:
```suggestion
- The output from the <a href="/assess-api#/operations/get-data-companies-companyId-connections-connectionId-assess-accounts-categories">List suggested and confirmed account categories</a> endpoint.
```

File: docs/auth-flow/customize/set-up-redirects.md [hunk 3]
```diff
@@ -47,7 +47,12 @@ To set up a redirect with custom query parameters:
 
 It's not possible to specify default parameters. If you don't add a parameter to the Link URL when the redirect is built, it's replaced with an empty string.
 
-For example, if you set your Redirect URL to: `https://www.codat.io/{journeyType}/success?ClientType={clientType}`, then the link URLs would give the following outcomes:
+For example, if you set your Redirect URL to the URL below
```
Comment (d-laing) [inline docs/auth-flow/customize/set-up-redirects.md:50]: It's not best practice to divide a sentence up with a code block like this, especially when it comes to future translation. Please format like this:

> For example, if you set your Redirect URL to `https://www.codat.io/{journeyType}/success?ClientType={clientType}`, then the link URLs would give the following outcomes:

File: docs/auth-flow/customize/set-up-redirects.md [hunk 4]
```diff
@@ -73,8 +78,14 @@ Note that this page autosaves when changes are made.
 
 To set up a redirect with reserved query parameters:
 
-1. In the **Redirect URL** box, enter a base URL along with the reserved parameters you want to use to build the redirect. To add a parameter, wrap it in curly braces. For example: `https://redirect.site/{sourceType}/?flow=Codat&statuscode={statusCode}&errormessage={errorMessage}`
-2. If you use the redirect parameter values shown above, your customer is redirected to: `https://redirect.site/accounting/?flow=Codat&statuscode=403&errormessage=User%20cancelled`.
+1. In the **Redirect URL** box, enter a base URL along with the reserved parameters you want to use to build the redirect. To add a parameter, wrap it in curly braces. For example: 
+```
```
Comment (d-laing) [inline docs/auth-flow/customize/set-up-redirects.md:82]: Indent the backticks by three spaces - lines 82 and 83. This aligns them with the numbered steps.

File: docs/auth-flow/customize/set-up-redirects.md [hunk 5]
```diff
@@ -73,8 +78,14 @@ Note that this page autosaves when changes are made.
...
+https://redirect.site/{sourceType}/?flow=Codat&statuscode={statusCode}&errormessage={errorMessage}
+```
+2. If you use the redirect parameter values shown above, your customer is redirected to: 
+```
```
Comment (d-laing) [inline docs/auth-flow/customize/set-up-redirects.md:86]: Same point about spaces.

File: docs/auth-flow/customize/use-openid-connect.md [hunk 6]
```diff
@@ -25,11 +25,18 @@ This article explains how to configure your authorization process to use OpenID
 Update your redirect URL and Hosted Link URLs.
 
 1. Add any parameters that you require from the user profile to your [redirect URL]((/auth-flow/customize/set-up-redirects). For example:
-   `https://redirect.com/site/{companyId}?firstName={openId_given_name}&email={openId_email}&phone={openId_phone_number}`
+   ```
```
Comment (d-laing) [inline docs/auth-flow/customize/use-openid-connect.md:28]: http :)

File: docs/integrations/accounting/dynamics365businesscentral/accounting-dynamics365businesscentral-setup.md [hunk 7]
```diff
@@ -57,7 +57,7 @@ Register your Business Central application on the Microsoft Azure portal.
 
    - **Supported account types**: Select **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**. This allows Codat to access data in Dynamics 365 Business Central.
 
-   - **Redirect URI**: Select **Web**, then enter the following URL: `<https://dynamics365businesscentral.codat.io/oauth/callback`>
+   - **Redirect URI**: Select **Web**, then enter the following URL: ` <https://dynamics365businesscentral.codat.io/oauth/callback> `
```
Comment (d-laing) [inline docs/integrations/accounting/dynamics365businesscentral/accounting-dynamics365businesscentral-setup.md:60]:
```suggestion
   - **Redirect URI**: Select **Web**, then enter the following URL: `https://dynamics365businesscentral.codat.io/oauth/callback`
```

File: docs/integrations/accounting/quickbooksdesktop/software-and-hardware-requirements.md [hunk 8]
```diff
@@ -73,6 +73,6 @@ If you have a large number of companies, or particularly large company files, yo
 
 ## Firewall rules
 
-The QuickBooks Desktop connector communicates over port 443 to URLs hosted on `<https://quickbooksdesktop.codat.io`> in production.
+The QuickBooks Desktop connector communicates over port 443 to URLs hosted on <https://quickbooksdesktop.codat.io> in production.
```
Comment (d-laing) [inline docs/integrations/accounting/quickbooksdesktop/software-and-hardware-requirements.md:76]:
```suggestion
The QuickBooks Desktop connector communicates over port 443 to URLs hosted on `https://quickbooksdesktop.codat.io` in production.
```

File: docs/integrations/accounting/quickbooksdesktop/software-and-hardware-requirements.md [hunk 9]
```diff
@@ -73,6 +73,6 @@ If you have a large number of companies, or particularly large company files, yo
...
 
-If you experience problems with the connector transmitting data, please add the following URL to your firewall allow list: `<https://quickbooksdesktop.codat.io/`>
+If you experience problems with the connector transmitting data, please add the following URL to your firewall allow list: <https://quickbooksdesktop.codat.io/>
```
Comment (d-laing) [inline docs/integrations/accounting/quickbooksdesktop/software-and-hardware-requirements.md:78]:
```suggestion
If you experience problems with the connector transmitting data, please add the following URL to your firewall allow list: `https://quickbooksdesktop.codat.io/`.
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Great job! I've made a few comments to address on some of the markup.  Thanks

**Comment by d-laing** [comment]
This should not be merged we're sure the Vercel build passes. Thanks!

File: docs/auth-flow/customize/use-openid-connect.md (hunk 6 above)
Comment (msmithcodat) [reply docs/auth-flow/customize/use-openid-connect.md]: updated :)

File: docs/auth-flow/customize/set-up-redirects.md (hunk 5 above)
Comment (msmithcodat) [reply docs/auth-flow/customize/set-up-redirects.md]: updated!

File: docs/auth-flow/customize/set-up-redirects.md (hunk 4 above)
Comment (msmithcodat) [reply docs/auth-flow/customize/set-up-redirects.md]: updated!

**Review by d-laing — APPROVED** [review]
Changes look good.

**Pull request merged**